L1Trigger/TrackFindingTMTT/interface/KFParamsComb.h

```diff
@@ -0,0 +1,41 @@
+///=== This is the Kalman Combinatorial Filter for 4 & 5 helix parameters track fit algorithm.
+
+#ifndef L1Trigger_TrackFindingTMTT_KFParamsComb_h
+#define L1Trigger_TrackFindingTMTT_KFParamsComb_h
+
+#include "L1Trigger/TrackFindingTMTT/interface/L1KalmanComb.h"
+#include <TMatrixD.h>
+#include "L1Trigger/TrackFindingTMTT/interface/L1track3D.h"
+
+namespace tmtt {
+
+  class KFParamsComb : public L1KalmanComb {
+  public:
+    enum PAR_IDS { INV2R, PHI0, T, Z0, D0 };
```

Comment:
This enum is used to address an array, so the integer values of the enum elements have importance. e.g. See https://github.com/cms-L1TK/cmssw/blob/L1TK-integration-TMTT-pre5/L1Trigger/TrackFindingTMTT/src/KFParamsComb.cc#L107 . (In the new code, this happens even more, since an earlier comment advised to stop using map<string,thing> and instead to use vector<thing> addressed by enum. 
If we change to using "enum class", then every single time we use the enums in this way (and there are a few dozen), we would be forced to use the very long "static_cast<int>(PAR_IDS:PHI0)" instead of the much shorter "PHI0" etc.